CHANGELOG.md

```diff
@@ -92,7 +92,7 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 - In the context of a systematic literature review (SLR), we reworked the "Define study" parameters dialog. [#9123](https://github.com/JabRef/jabref/pull/9123)
 - We upgraded to Lucene 9.4 for the fulltext search. The search index will be rebuild. [#9213](https://github.com/JabRef/jabref/pull/9213)
 - We disabled the "change case" menu for empty fields. [#9214](https://github.com/JabRef/jabref/issues/9214)
-- We disabled the conversion menu for empty fields. [#9200](https://github.com/JabRef/jabref/issues/9200)
+- We disabled the conversion menu for empty fields. [#9200](https://github.com/JabRef/jabref/issues/9200) 
```

Comment:
Why is there an extra white space here?

```suggestion
- We disabled the conversion menu for empty fields. [#9200](https://github.com/JabRef/jabref/issues/9200)
```

Reply:
Got it @k3KAW8Pnf7mkmdSMPHz27, removed the white space.